### ISSUE TITLE
UU: Fix styling UxSignalsWidget

### ISSUE THREAD

**Issue by PerOlavM** [navikt/nav-enonicxp-frontend#2024]
Flere WCAG-brudd i denne widgeten.
https://my2.siteimprove.com/Inspector/1002489/A11Y/Page?pageId=50561017192&impmd=F9AEE0169DCE337D2604EA436A02A736&conf=a+aa+aaa+aria+si&decision=dismissed+falsePositive+none#/sia-r74/failed/MWpUL0FQTFA=;ZWMwNDg4ZGI=
